app/build.gradle

```diff
@@ -9,7 +9,7 @@ android {
     buildToolsVersion "29.0.2"
     defaultConfig {
         applicationId "com.scc.bukusakuonline"
-        minSdkVersion 15
```

Comment:
gasah diganti , aku ada sdk yang dibawah 23

Reply:
sama file2 yang double tolong hapusin nanti menuh2 in repo. jangan ikut dipush juga